osschallenge/settings.py

```diff
@@ -181,7 +181,7 @@
 DEFAULT_FROM_EMAIL = env.str("DEFAULT_FROM_EMAIL", default="osschallenge@oss-challenge.ch")
 
 
-STATIC_ROOT = os.path.join(BASE_DIR, "osschallenge/static")
+STATIC_ROOT = "/static"
```

Comment:
I don't think it's a good idea to put it in the root folder. Why change it? I think the only fix needed was the `collectstatic` call